addons/website_payment/static/src/js/wysiwyg.js

```diff
@@ -0,0 +1,19 @@
+/** @odoo-module **/
+
+import Wysiwyg from "web_editor.wysiwyg";
+
+Wysiwyg.include({
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+    /**
+     * @override
+     */
+    removeLink() {
+        if (this.lastElement.classList.contains("s_donation_donate_btn")) {
```

Comment:
Same way here as above, what is the use of this method? if it is useful then add document string.